### PR TITLE
PL Launch 2023 - Homepage hero banner

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add a superhero theme hero banner on the https://code.org homepage for the 2023 Professional Learning launch.
- Uses the same superhero imagery from the superhero skinny banners (see related PR below)
- Uses the DCDO flag `pl-launch-hero-banner`

**Related PRs:** 
- https://github.com/code-dot-org/code-dot-org/pull/49556
- https://github.com/code-dot-org/code-dot-org/pull/51356

**Asana task:** https://app.asana.com/0/1202773141567416/1203483189611897/f

----

### Desktop
Screenshot 2023-01-06 at 11 01 13 AM

### Tablet
Tablet

### Mobile
Mobile

### CODE REVIEW

**Comment by dmcavoy** [comment]
The screenshots look great! I will leave it up to engineering to review the code.

File: pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    &.image-tablet {
+      display: none;
+    }
+
+    @media screen and (max-width: $width-md) {
+      width: 850px;
+      right: -25em;
+
+      &.image {
+        display: none;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss:78]: Hiding the desktop image on tablet

File: pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      right: -25em;
+
+      &.image {
+        display: block;
+        width: 100%;
+        left: 4em;
+      }
+
+      &.image-tablet {
+        display: none;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss:99]: Hiding the tablet image on mobile; this uses the desktop image file

File: pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  }
+
+  img {
+    position: absolute;
+    bottom: 0;
+    z-index: -1;
+    width: 1080px;
+
+    &.image-tablet {
+      display: none;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/pl-hero-banner.scss:70]: Hiding the tablet image on default

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 4]
```diff
@@ -52,6 +52,8 @@
           = I18n.t(entry[:text])
       - elsif entry[:type] == "image"
         %img.image{src: entry[:url]}
+      - elsif entry[:type] == "image-tablet"
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:55]: Added a tablet image option for use on homepage.json: https://github.com/code-dot-org/code-dot-org/blob/ea32e656df3fef6cde8f5090039d75210d2394f7/pegasus/sites.v3/code.org/homepage.json#L93-L94

**Review by TurnerRiley — APPROVED** [review]
Looks great to me!

**Review by davidsbailey — APPROVED** [review]

**Pull request merged**